index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Amatic+SC&family=Raleway&family=Rampart+One&family=Roboto&display=swap"
+    rel="stylesheet">
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+  <header class="header">
+    <!-- Some kind of logo and navbar -->
+    <!--./help the website to find the path-->
+    <div class="logo"> <img src="./pickle-logo.webp" alt="logo"><a href="#"></a> </div>
+
+
+
+
+    <nav class="navigation">
+      <a href="#">Pickle time</a>
+      <a href="#">Vegetables</a>
+      <a href="#">Seasons</a>
+      <a href="#">Contact</a>
+
+    </nav>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  </header>
+
+  <section class="big-news">
+    <!-- The big news content -->
+
+    <div>
+      <img class="big-news-picture"
+        src="https://www.liveeatlearn.com/wp-content/uploads/2020/03/quick-pickled-vegetables-9-650x976.jpg.webp"
+        alt="pickles">
+    </div>
+
+    <div class="big-news-text">
+      <div class="animation">
+        <h1> Are you in a pickle? </h1>
+      </div>
+
+      <p>HODOR hodor, hodor. Hodor... Hodor hodor. HODOR hodor, hodor. Hodor hodor. HODOR?
+
+        Hodor, hodor. Hodor. Hodor. HODOR? Hodor hodor. Hodor... Hodor hodor. Hodor hodor HODOR! Hodor. HODOR? Hodor...
+        Hodor hodor.HODOR hodor, hodor. Hodor... Hodor hodor. HODOR hodor, hodor. Hodor hodor. HODOR?
+
+        Hodor, hodor. Hodor. Hodor. HODOR? Hodor hodor. Hodor... Hodor hodor. Hodor hodor HODOR! Hodor. HODOR? Hodor...
+        Hodor hodor.HODOR hodor, hodor. Hodor... Hodor hodor. HODOR hodor, hodor. Hodor hodor. HODOR?
+
+        .</p>
+    </div>
```

Comment:
I think there is too much text in this paragraph section. I'm not sure if that is a style comment or an HTML comment. Or maybe both. It does, however affect the styling of the page when viewed on a smaller screen.